webrtc.html

```diff
@@ -1297,6 +1299,25 @@
               </li>
 
               <li>
+                <p>If <var>candidate</var> is <code>null</code>, then run the
+                following steps:</p>
+                <ol>
+                  <li>
+                    <p>For each media section in the last successfully applied
+                    remote description, perform the processing for an
+                    end-of-candidates indication for said media section as
+                    defined in [[TRICKLE-ICE]], unless an ICE restart occurred
+                    for said media section since setting the last remote
+                    description.</p>
```

Comment:
Perhaps a better way of saying this is:  We need to make it clear what happens when this is done:

pc.setRemoteDescription(remote_offer_with_ice_restart);
pc.addIceCandidate(null);
pc.setRemoteDescription({type: rollback});

or this:

pc.setLocalDescription(local_offer_with_ice_restart);
pc.addIceCandidate(null);
pc.setLocalDescription({type: rollback});

Can the pc now go to completed/failed?